libraries/resource_systemd_logind.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'resource_systemd_daemon'
+require_relative 'systemd_logind'
+
+class Chef::Resource
+  class SystemdLogind < Chef::Resource::SystemdDaemon
+    self.resource_name = :systemd_logind
+    provides :systemd_logind
+
+    def daemon_type(_ = nil)
```

Comment:
it's an override for the [default in the base class](https://github.com/nathwill/chef-systemd/blob/master/libraries/resource_systemd_daemon.rb#L13), analogous to the [unit_type in systemd_unit](https://github.com/nathwill/chef-systemd/blob/master/libraries/resource_systemd_unit.rb#L17), used in each descendant from the base class (e.g. [systemd_timesyncd](https://github.com/nathwill/chef-systemd/blob/master/libraries/resource_systemd_timesyncd.rb#L10)); ultimately it's used to determine the [config path](https://github.com/nathwill/chef-systemd/blob/master/libraries/helpers.rb#L47) (i.e. systemd-journald => journald.conf, systemd-timesyncd => timesyncd.conf). i suppose we could derive it from the class name instead, but this seemed more straightforward at the time, and there was the precedent with unit_type, so i went with it. if you've got a better idea, i'd love to hear it!